frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsHours/FeatureMetricsHours.tsx

```diff
@@ -30,9 +31,11 @@ export const FeatureMetricsHours = ({
         });
     };
     const { isEnterprise } = useUiConfig();
-    const options = isEnterprise()
-        ? [...hourOptions, ...daysOptions]
-        : hourOptions;
+    const extendedUsageMetricsEnabled = useUiFlag('extendedUsageMetrics');
+    const options =
+        isEnterprise() && extendedUsageMetricsEnabled
```

Comment:
enterprise customers should still see 30 days even without extended usage metrics. extended is for 90 days

Reply:
Recap: After some testing and validation we realized that anything > 48h is ignored backend when flag isn't set, so we'll go ahead with this PR as is